packages/cli/src/lib/container/streamer.ts

```diff
@@ -1,17 +1,16 @@
-const http = require('http-call').HTTP
-const maxRetries = 30
-
-async function call(url, out, retries) {
+async function call(url: string, out: NodeJS.WriteStream, retries: number) {
+  const http = require('http-call').HTTP
+  const maxRetries = 30
   try {
-    let {response} = await http.stream(url)
-    response.on('data', function (d) {
+    const {response} = await http.stream(url)
+    response.on('data', function (d: string) {
       out.write(d)
     })
     return await new Promise(function (resolve, reject) {
       response.on('error', reject)
       response.on('end', resolve)
     })
-  } catch (error) {
+  } catch (error: any) {
     if (error.statusCode === 404 && retries <= maxRetries) {
       return new Promise(function (resolve, reject) {
         setTimeout(function () {
```

Comment:
optional small tweak to avoid the use of `any`
```suggestion
  } catch (error) {
    const {statusCode} = error as {statusCode: number}
    if (statusCode === 404 && retries <= maxRetries) {
      return new Promise(function (resolve, reject) {
        setTimeout(function () {
```